packages/dapp/src/pages/Main.tsx

```diff
@@ -1,36 +1,125 @@
-import { Box, Text } from 'grommet';
+import { useState, useEffect } from 'react';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Text,
+  TextInput,
+} from 'grommet';
+import { encrypt, decrypt } from '../utils/crypto';
+import {useAppState} from "../store";
 
-import { useAppState } from '../store';
-import { SignInButton, SignOutButton } from '../components/buttons/web3Modal';
-import { Account } from '../components/Account';
+export const Main = () => {
+  const { encryptionKey } = useAppState();
+  const { encryptedData, switchEncryptedData } = useAppState();
```

Comment:
the same as in the https://github.com/windingtree/org.id-profile-manager/pull/3/files#r796513902